app/cli/cmd/workflow_contract_list.go

```diff
@@ -50,7 +50,12 @@ func contractListTableOutput(contracts []*action.WorkflowContractItem) error {
 
 	t.AppendHeader(table.Row{"Name", "Latest Revision", "Created At", "# Workflows"})
 	for _, p := range contracts {
-		t.AppendRow(table.Row{p.Name, p.LatestRevision, p.CreatedAt.Format(time.RFC822), len(p.Workflows)})
+		wfLen := len(p.Workflows)
```

Comment:
I do not think the client needs to take into account both modes, it is released together (client and server), you need to think more about previous clients.